var/spack/repos/builtin/packages/libcxi/package.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Libcxi(AutotoolsPackage):
+    """The CXI library provides interfaces which interact directly with CXI
+    drivers."""
+
+    homepage = "https://github.com/HewlettPackard/shs-libcxi"
+    git = "https://github.com/HewlettPackard/shs-libcxi.git"
+
+    license("LGPL-2.1-or-later or BSD-3-Clause")
+
+    version("main", branch="main")
+
+    variant("oneapi", default=False, description="Build with OneAPI Level Zero support")
```

Comment:
```suggestion
    variant("level_zero", default=False, description="Build with OneAPI Level Zero support")
```

this is the common name of the variant since last week #47985

Reply:
Fixed.